descope/api/client.go

```diff
@@ -101,6 +101,7 @@ var (
 			ssoApplicationLoad:                 "mgmt/sso/idp/app/load",
 			ssoApplicationLoadAll:              "mgmt/sso/idp/apps/load",
 			userCreate:                         "mgmt/user/create",
+			testUserCreate:                     "mgmt/user/create/test",
```

Comment:
Why not using the `mgmt/user/create` with test? as today? what has changed?

Reply:
We added the mgmt key role User-Testing and therefor we added new API that will require only this role.